src/App.tsx

```diff
@@ -1,14 +1,72 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [filterByTitle, setFilterByTitle] = useState<string>('');
+  const [filter, setFilter] = useState<string>('');
+
+  const selectTodos = (todo: Todo) => {
+    const findTodo = todos.find(value => value === todo) as Todo;
```

Comment:
```suggestion
    const findTodo = todos.find(value => value === todo) as Todo;
```
there is something wrong with typing, you shouldn't have to use `as`